leader-election-impl/src/main/java/com/palantir/paxos/SqlitePaxosStateLog.java

```diff
@@ -27,91 +28,109 @@
 import org.jdbi.v3.sqlobject.SqlObjectPlugin;
 import org.jdbi.v3.sqlobject.customizer.Bind;
 import org.jdbi.v3.sqlobject.customizer.BindPojo;
-import org.jdbi.v3.sqlobject.customizer.Define;
 import org.jdbi.v3.sqlobject.statement.SqlBatch;
 import org.jdbi.v3.sqlobject.statement.SqlQuery;
 import org.jdbi.v3.sqlobject.statement.SqlUpdate;
 
 import com.palantir.common.persist.Persistable;
 
 public final class SqlitePaxosStateLog<V extends Persistable & Versionable> implements PaxosStateLog<V> {
-    private final String namespace;
+    private final Client namespace;
+    private final String seqId;
```

Comment:
For my knowledge, what is seqId?

Reply:
Yes, this is not super descriptive, I imagine it's a combination of use case and acceptor/learner?

Reply:
Right, yeah. The problem is that `PaxosUseCase` as a concept only exists in TimeLock (this project is also used by embedded services and clustered lock in large internal product), which is why I opted to be generic and just describe it as a sequence ID. It _is_ meant to be `(PaxosUseCase, Acceptor|Learner)`, yes.

Reply:
I think we discussed offline and went for `useCase` - this does run the risk of people conflating it with `PaxosUseCase`, but it is general enough still.